src/onemod/pipeline.py

```diff
@@ -260,6 +263,28 @@ def save_validation_report(
         report_path = validation_dir / "validation_report.json"
         serialize(collector.errors, report_path)  # type: ignore[arg-type]
 
+    def add_tasks_to_workflow(
```

Comment:
Other people can chime in, but since we tried to keep the orchestration separate from the pipeline/stage classes, I'm wondering if this function belongs in the jobmon backend rather than the pipeline class.